src/dsp_tools/models/exceptions.py

```diff
@@ -64,7 +65,56 @@ def __str__(self) -> str:
 
 class InternalError(BaseError):
     """
-    Class for errors that will be handled by a higher level function
+    Class for errors that are raised if the user cannot solve the problem themselves.
+    """
+
+    def __init__(self, custom_msg: str | None = None, keep_default_msg: bool = True) -> None:
+        if not keep_default_msg and custom_msg:
+            super().__init__(custom_msg)
+        else:
+            default_msg = (
+                "\n\nAn internal error occurred.\n"
+                "Please contact the dsp-tools development team with the following information:\n"
+                "\t- Which command was used.\n"
+                "\t- If applicable, any files that were used in conjunction with the command.\n"
+                "\t- A file with the terminal output copied into.\n"
+                f"\t- The log files called 'logging.log', if there are several, include all.\n"
+                f"\t  They can be found at: {Path.home() / Path('.dsp-tools')}\n"
+            )
+            if custom_msg:
+                default_msg = "\n\n" + custom_msg + "\n--------------------------" + default_msg
+            super().__init__(default_msg)
```

Comment:
I think this could be streamlined:

```suggestion
        default_msg = ...
        match keep_default_ms, custom_msg:
            case False, str():
                super().__init__(custom_msg)
            case _, None:
                super().__init__(default_msg)
            case _:
                 super().__init__(f"{custom_msg}\n--------------------------\n{default_msg}")
```

... or around those lines - it's not exactly identical now. But I personally find it easier to grasp the logic like this.  
(also, I think the concatenated message in your version may have the line breaks in the wrong spot)

Reply:
I see now - I think I missed the fact that default_msg starts with linebreaks